dex/fiatrates/fiatrates.go

```diff
@@ -39,6 +39,8 @@ func parseCoinpapNameSymbol(name, symbol string) (string, string) {
 	case "polygon":
 		symbol = "matic"
 		name = "polygon"
+	case "weth":
+		name = "weth"
 	}
```

Comment:
Also, set `symbol` to `weth`.

```suggestion
name = "weth"
symbol = "weth"
}
```

Reply:
`symbol` is already "weth".

Reply:
Oh yeah. Had a glance.